packages/preset-icons/src/types.ts

```diff
@@ -4,7 +4,7 @@ import type { Awaitable, CSSObject } from '@unocss/core'
 
 interface IconMeta {
   collection: string
-  icon: string
+  name: string
```

Comment:
is this a breaking change?

Reply:
In the #3774, I added handling of icon meta for process, but in this PR, referring to the previous variable naming, I think `name` is a better choice.
https://github.com/unocss/unocss/blob/31e6709c43068a3ff216c36c669007ab38fb33d3/packages/preset-icons/src/core.ts#L73-L75

Or I can revert the changes here.

Reply:
We should avoid breaking changes.

I won't be available until next Tuesday.